Content.Server/Body/Systems/BodySystem.cs

```diff
@@ -119,14 +118,15 @@ public override bool DropPart(EntityUid? partId, BodyPartComponent? part = null)
     protected override void InitBody(BodyComponent body, BodyPrototype prototype)
     {
         var root = prototype.Slots[prototype.Root];
+        Containers.EnsureContainer<Container>(body.Owner, BodyContainerId);
+        if (root.Part == null)
+            return;
         var bodyId = Spawn(root.Part, body.Owner.ToCoordinates());
         var partComponent = Comp<BodyPartComponent>(bodyId);
-        var slot = new BodyPartSlot(root.Part, body.Owner, partComponent.PartType);
+        var slot = new BodyPartSlot("root", body.Owner, partComponent.PartType);
```

Comment:
You can still use root.Part here if you are checking for null, unless you want the id to be "root" for some other reason